### ISSUE TITLE
Change term - subgenus

### ISSUE THREAD

**Issue by tucotuco** [tdwg/dwc#320]
## Change term

* Submitter: John Wieczorek @tucotuco 
* Justification (why is this change necessary?): Based on discussions in Issue #176, it was suggested that the examples for subgenus were not strictly correct. The examples were changed to remove the reference to the genus, but the definition of the term still includes "Values should include the genus to avoid homonym confusion." First of all, that should be a usage note and not a part of the definition and should have been pulled when work was done to divide comments from definitions. This is an interesting case, because it isn't actually a semantic change that is being proposed, despite the fact that the definition would change. Thus, it is a little unclear whether it would have to go through public review. To stay on the safe side, we should include it with a series of Taxon term change requests. The [issue that inspired this change request](https://github.com/tdwg/dwc/issues/319) was made by @Jegelewicz.
* Proponents (who needs this change): Everyone sharing or searching for information based on subgenus.

Current Term definition: https://dwc.tdwg.org/terms/#dwc:subgenus

Proposed new attributes of the term:

* Term name (in lowerCamelCase): subgenus
* Organized in Class (e.g. Location, Taxon): Taxon
* Definition of the term: **The full scientific name of the subgenus in which the taxon is classified.** 
* Usage comments (recommendations regarding content, etc.): **Values should not include the genus, nor should they be in parentheses.**
* Examples: `Strobus`, `Amerigo`, `Pilosella`
* Refines (identifier of the broader term this term refines, if applicable):
* Replaces (identifier of the existing term that would be deprecated and replaced by this term, if applicable): http://rs.tdwg.org/dwc/terms/version/subgenus-2017-10-06
* ABCD 2.06 (XPATH of the equivalent term in ABCD or EFG, if applicable): DataSets/DataSet/Units/Unit/Identifications/Identification/Result/TaxonIdentified/HigherTaxa/HigherTaxon[HigherTaxonRank="subgenus"]/HigherTaxonName

**Comment by baskaufs** [comment]
`dwc:subgenus` falls into the category of "[convenience terms](https://dwc.tdwg.org/rdf/#27-darwin-core-convenience-terms-non-normative)" so there will be no change to a `dwciri:` analog.

**Comment by nielsklazenga** [comment]
I would remove the ABCD 2.06 equivalent, as Subgenus in ABCD is the `infragenericEpithet`, not the taxon as intended here.

**Comment by nielsklazenga** [comment]
ABCD 2.06 has HigherTaxa (and HigherTaxon), each with HigherTaxonName and HigherTaxonRank, so no equivalent for `subfamily` or any of the other similar terms.

**Comment by tucotuco** [comment]
> I would remove the ABCD 2.06 equivalent, as Subgenus in ABCD is the `infragenericEpithet`, not the taxon as intended here.

@nielsklazenga I'm not sure I understand this, but I trust your observation and no one has objected. I have amended the ABCD mapping from "DataSets/DataSet/Units/Unit/Identifications/Identification/TaxonIdentified/ScientificName/NameAtomised/Zoological/Subgenus" to "not in ABCD".

**Comment by nielsklazenga** [comment]
@tucotuco , `kingdom`, `phylum`, `...`, and `subgenus` are all taxa (or at least complete names), while `genericName`, `infragenericEpithet`, `specificEpithet` and `infraspecificEpithet` are parts of names (that are combinations). The `../Subgenus` elements in ABCD are equivalent to `infragenericEpithet` (#30), not `subgenus`.

**Comment by tucotuco** [comment]
Thank you @nielsklazenga. I think that clarification will be helpful for a lot of people.

**Comment by nielsklazenga** [comment]
I did not think about this before, but even simpler might be that `subgenus` is part of `higherClassification` and `infragenericEpithet` of `scientificName`.


ABCD 2.06 equivalent for `subgenus` could be 
`DataSets/DataSet/Units/Unit/Identifications/Identification/Result/TaxonIdentified/HigherTaxa/HigherTaxon[HigherTaxonRank="subgenus"]/HigherTaxonName`, if you want to go that way. I think that is valid XPath, but my XML is a bit rusty.

**Comment by jholetschek** [comment]
That's a good idea, Niels, but according to the ABCD documentation, the HigherTaxa are for "taxonomic ranks above the rank of genus". So I'm afraid there is no equivalent.

**Comment by nielsklazenga** [comment]
Ha, good one @jholetschek. Could you confirm that I at least put the condition in the XPath correctly? @DavidFichtmueller suggested something similar for `subfamily` in https://github.com/tdwg/dwc/issues/44#issuecomment-831194476.

**Comment by jholetschek** [comment]
Yes, David just confirmed that this XPath is correct ;)

**Comment by tucotuco** [comment]
Thanks @nielsklazenga and @jholetschek, updated in the proposal in the first comment.

**Comment by JCGiron** [comment]
I would argue that keeping the subgenus accompanied by its genus and in parenthesis [i.e. Genus (Subgenus)] would prevent misinterpretations. In some groups of beetles, sometimes, entire subgenera have been transferred to different genera, so I think it is important to be able to point to the correct, current treatment and avoid ambiguity in those cases.

**Comment by tucotuco** [comment]
This proposal has been labeled as 'Controversial'. It will remain open for public review in pursuit of a consensus solution for another 30 days, but will not be included in the release to be prepared from the public review of 2021-05-01/2021-05/31.

**Comment by dschigel** [comment]
As this came in the GBIF Global Nodes meeting, my two cents from entomological past: subgenus has been a useful and broadly used rank in families with huge genera, e.g. [Staphylinidae](http://spongymesophyll.com/Herman_2001_Catalog_of_the_Staphylinidae_Part_1.pdf). I have seen independent use of subgeneric epithets but I can consider this suboptimal practice and I support use of sg together with the genus epithet. Example of the view that is pleasing to my eyes from page 5: _Acylophorus_ (_Paracylophorus_) _schmidti_ Bierig, 1938. Similar wishes were expressed above.

**Comment by tucotuco** [comment]
Public review of this issue has now concluded with objections to the proposed change. The issue will remain open for discussion and potential resolution.

**Comment by nielsklazenga** [comment]
I agree with @JCGiron and @dschigel that subgenus should include the generic name. The Botanical Code requires this in infrageneric names, so for me it does not need to be in the definition, but it is probably different in other Codes.

The botanical examples should be written as 'Pinus subgen. Strobus' and 'Hieracium subgen. Pilosella', but they are problematic, as these names do not exist. "Strobus" is a section of _Pinus_ (or a genus on its own) and _Pilosella_ is a genus, not a subdivision of _Hieracium_. It might be best to remove the botanical examples altogether, as the term is unlikely to be used by botanists, as in botany there are additional infrageneric ranks.

I think the addition of the term `infragenericEpithet` (#30) has achieved what the proposers tried to achieve here, so I think this issue can be closed, but do not take my word for it.

There remains the issue that currently the examples for [`subgenus`](https://dwc.tdwg.org/terms/#dwc:subgenus) do not correspond with the definition.

**Comment by timrobertson100** [comment]
> I think the addition of the term infragenericEpithet (#30) has achieved what the proposers tried to achieve here

Please bear in mind I'm not a biologist when reading this, but I'm not sure I agree.

Using the scientific name  **_Doryteuthis (Amerigo) surinamensis_ (Voss, 1974)** as an example, you would have _Amerigo_ as the infragenericEpithet, but that is different from our having the ability to have a concept for the subgenus which would presumably be something like **_Doryteuthis (Amerigo)_ Brakoniecki, 1996** 

> keeping the subgenus accompanied by its genus and in parenthesis [i.e. Genus (Subgenus)] would prevent misinterpretations

With guidance to render it as you propose, is it perhaps disambiguated enough to avoid misuse? 

I'd offer that homonyms are everywhere in tree hierarchies - e.g. `city=Newcastle`, `genus=Oenanthe` - but that doesn't stop having a term to refer to the notion of a `city` or a `genus` and using available context to disambiguate. 

I'm not the instigator of this request, but my interest in keeping the discussion open is that, rightly or wrongly, GBIF uses convenience denormalized pointers into a taxonomy to search occurrence records. My concern is that without a subgenus and its associated `subGenusKey` it _may_ be difficult for us to provide filtering and reporting.

**Comment by mdoering** [comment]
It would be good to at least address the wrong examples for the current definition which all uses just the plain subgenus name without genus or authorship

**Comment by mdoering** [comment]
This is very much related to all other higher rank terms which all say the same, e.g. for family:

> The full scientific name of the family in which the taxon is classified.
Examples | Felidae, Monocleaceae

So does the full scientific name include authorship? The DwC definition of dwc:scientificName suggests that:
> The full scientific name, with authorship and date information if known

And if that's the case I reckon all examples should at least also include one with authorship. And probably the usage notes should be emended too.

**Comment by nielsklazenga** [comment]
@timrobertson100 :

> Please bear in mind I'm not a biologist when reading this, but I'm not sure I agree.

I am a botanist, which probably disqualifies me from talking about `subgenus`. I did realise something though when reading your next paragraph. At least in botany, `infragenericEpithet` is only used in infrageneric names (names for subgenera, sections etc.), so `infragenericEpithet` cannot be used instead of `subgenus`, which is often used with species and infraspecific taxa.

**Comment by nielsklazenga** [comment]
@mdoering :

> So does the full scientific name include authorship?

At least according to the Botanical Code (sorry, it is the only one I know) the authorship is not part of the scientific name. I do not think the definition of `dwc:scientificName` suggests that it does, but I will grant you that it can be read either way.

I always get tripped up by 'full scientific name', as to me, for taxa below the rank of genus, that means the generic name plus the epithets and rank prefixes. It might mean something different to other people.

**Comment by Jegelewicz** [comment]
dwc:scientificName 

> The full scientific name, with authorship and date information if known. 

Seems to me that authorship is part of that? If authorship is not included, there could be added confusion when it comes homonyms, although this could be disambiguated with scientificNameAuthorship (which ends up seeming redundant if it is also part of scientificName). So there we are - maybe scientificName should NOT include authorship, then we could lose GBIF's canonicalName?

**Comment by nielsklazenga** [comment]
@Jegelewicz , I agree, but it is bound to be controversial and I am not sure it is a fight worth having. I think it is probably best to aim for the inclusion of authorship and date to be optional, which it sort of is now, and leave whether it SHOULD or SHOULD NOT be included to application profiles. canonicalName is a useful term no matter what.

**Comment by tucotuco** [comment]
Following up on the current status of this issue, it remains "controversial". There are unresolved arguments about the proposed changes even though there seems to be agreement that there should be a change.

**Comment by mdoering** [comment]
I think the definition and ABCD change is uncontroversial.
The open questions are whether the usage comments & examples should recommend the genus to be included or not. After reading the entire conversation here again I feel we mostly prefer the genus to be included, but it is also subject to the individual code. So maybe we do not even need to be more specific?

>Usage comments (recommendations regarding content, etc.): Values should not include the genus, nor should they be in parentheses.
>Examples: Strobus, Amerigo, Pilosella

How about:
>Definition of the term: The full scientific name of the subgenus in which the taxon is classified.
>Usage comments (recommendations regarding content, etc.): Values should be the complete subgenus name as required by the respective nomenclatural code.
>Examples: Abacetus (Parastygis); Dicranum subgen. Orthodicranum

**Comment by mdoering** [comment]
I think there is no major objection to the summary I gave above, so I recommend it for executive review.
The authorship is missing in all existing examples for higher taxa like [dwc:order](https://dwc.tdwg.org/terms/#dwc:order), family and so on, so I think it is wise to also not include it here in examples.

**Comment by dschigel** [comment]
Saw you latest post. Example - I used this as a nomenclator in my beetle-fungus studies in Finland, you can see family authors, and many sg. cases, if it helps https://tietopankki.luomus.fi/wp-content/uploads/2023/02/Silfverberg_Enumeratio_Coleopterorum_Sahlbergia-9-1-2004.pdf. Note however, that Hans is one of the most careful and detailed nomenclature experts I ever met, so maybe this is not very representative
